client/client/modules/render/whole-genome-render/partials/chromosomeColumn/chromosomeColumnRenderer.js

```diff
@@ -0,0 +1,108 @@
+import { drawingConfiguration } from '../../../core';
+
+export class ChromosomeColumnRenderer {
+
+    constructor(container, canvasSize, drawingConfig, chromosomes, maxChrSize){
+      
+        Object.assign(this, {
+            container,
+            canvasSize,
+            drawingConfig,
```

Comment:
Here we use `drawingConfig` that was imported from *whole-genome-config.js* file and passed to the `chromosomeColumnRenderer` as a constructor argument; though it's working now, it's not completely safe to access the properties like `drawingConfig.start.topMargin` (e.g. if `chromosomeColumnRenderer` was initialized without `drawingConfig`).
Let's implement the *merging* of the passed in config with the initial one to prevent possible `Cannot read someProperty of undefined` errors.